SU2_CFD/src/output/output_structure_legacy.cpp

```diff
@@ -360,7 +360,7 @@ void COutputLegacy::SetConvHistory_Header(ofstream *ConvHist_file, CConfig *conf
 
   bool output_surface = (config->GetnMarker_Analyze() != 0);
   bool output_comboObj = (config->GetnObj() > 1);
-  bool output_per_surface = config->GetWrt_Surface();
+  bool output_per_surface = true;
```

Comment:
Obviously this kind of change (there are just 3 in this file) takes away functionality from users of output_structure_legacy. In these cases I was always generous 👑  and enabled stuff which potentially clutters output but I am unsure whether anybody even uses this at this point

Reply:
EDIT: now it is just WRT_SURFACE